qiita_db/metadata_template/base_metadata_template.py

```diff
@@ -1083,3 +1083,30 @@ def categories(self):
                 cols[idx] = self.translate_cols_dict[c]
 
         return cols
+
+    def update_category(self, category, samples_and_values):
+        """Update an existing column
+
+        Parameters
+        ----------
+        category : str
+            The category to update
+        samples_and_values : dict
+            A mapping of {sample_id: value}
+
+        Raises
+        ------
+        QiitaDBUnknownIDError
+            If a sample_id is included in values that is not in the template
+        QiitaDBColumnError
+            If the column does not exist in the table. This is implicit, and
+            can be thrown by the contained Samples.
+        """
+        if not set(self.keys()).issuperset(samples_and_values):
+            missing = set(self.keys()) - set(samples_and_values)
+            table_name = self._table_name(self.study_id)
+            raise QiitaDBUnknownIDError(missing, table_name)
+
+        for k, v in viewitems(samples_and_values):
+            sample = self[k]
+            sample[category] = v
```

Comment:
You're not saving the instantiation, it is happening at the moment that you do `self[k]`. Anyways, this is left like this because I modify this in my other PR #1054....